### DISCUSSION TITLE
BREAKING CHANGE: webpack < 5 used to include polyfills for node.js core modules by default.

### DISCUSSION THREAD

**Discussion by jdeola** [gatsbyjs/gatsby#34916] [Help]
Hello,

I'm a relatively new developer so forgive me if this issue is missing context, but I recently updated gatsby-CLI and installed dotenv in my package.json, and now when I run gatsby develop, I keep getting this error for MANY of the files found in node_modules. 

I'm unsure where to even begin debugging this, as everything was working fine a few days ago prior to updating. Has anyone else come across this issue? I'm unsure which package/plugin revert to a prior version to fix this. 

Here are the deps in package.json:

`  "dependencies": {
    "@fortawesome/fontawesome-svg-core": "^1.3.0",
    "@fortawesome/free-solid-svg-icons": "^6.0.0",
    "@fortawesome/react-fontawesome": "^0.1.17",
    "@reach/router": "^1.3.4",
    "@sendgrid/mail": "^7.6.1",
    "bootstrap": "^5.1.3",
    "gatsby": "^4.8.0",
    "gatsby-plugin-fontawesome-css": "^1.2.0",
    "gatsby-plugin-image": "^2.8.0",
    "gatsby-plugin-manifest": "^4.8.0",
    "gatsby-plugin-react-helmet": "^5.8.0",
    "gatsby-plugin-react-svg": "^3.1.0",
    "gatsby-plugin-sharp": "^4.8.0",
    "gatsby-plugin-utils": "^3.2.0",
    "gatsby-remark-images": "^6.8.0",
    "gatsby-remark-relative-images": "^2.0.2",
    "gatsby-source-filesystem": "^4.8.0",
    "gatsby-transformer-sharp": "^4.8.0",
    "prop-types": "^15.8.1",
    "react": "^17.0.2",
    "react-bootstrap": "^2.1.2",
    "react-dom": "^17.0.2",
    "react-helmet": "^6.1.0"
  },
  "devDependencies": {
    "dotenv": "^16.0.0",
    "prettier": "^2.5.1"
  },`

Thank you in advance for any help!

EDIT: i seem to have solved the issue by running the command:

`npm install gatsby@next --legacy-peer-deps`

EDIT 2: I'm having the same issue again...

**Comment by tyhopp** [comment]
Hi @jdeola, have you tried creating a [minimal reproduction][reproduction] for yourself to see if the same issue happens? Another useful thing to try is to use `npx gatsby new` to make sure there isn't an issue with your global install.

[reproduction]: https://gatsby.dev/reproduction

**Comment by jdeola** [comment]
Thank you for the suggestions! I will try them both
